frontend/src/metabase/nav/components/ProfileLink/useHelpLink.ts

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+import { UtilApi } from "metabase/services";
+import { useSelector } from "metabase/lib/redux";
+import { getSetting, getIsPaidPlan } from "metabase/selectors/settings";
+import { getUser } from "../../../selectors/user";
+
+export const useHelpLink = (): { visible: boolean; href: string } => {
+  const helpLinkSetting = useSelector(state => getSetting(state, "help-link"));
```

Comment:
you reference `helpLinkType` in `enterprise/frontend/src/metabase-enterprise/whitelabel/components/HelpLinkSettings/HelpLinkRadio.tsx`. What do you think about using the same variable name then?